test/models/student_test.rb

```diff
@@ -1,7 +1,26 @@
 require 'test_helper'
 
 class StudentTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  def setup
+    @user = users(:one)
+    @s = students(:one)
+    # @s = Student.new(class_rank: "junior", room_draw_number: 53)
+  end
+
+
+  test "should be valid" do
+    assert @s.valid?, @s.errors.full_messages
+  end
+
+  test "class_rank should be present" do
+    @s.class_rank = nil
+    assert_not @s.valid?
+  end
+
+  test "room_draw_number should be present" do
+    @s.room_draw_number = nil
+    assert_not @s.valid?
```

Comment:
This kind of test can also be done via [thoughtbot/shoulda](https://github.com/thoughtbot/shoulda) using `should validate_presence_of(:room_draw_number)`. This kind of library can help simplify various different kinds of tests; see the overview example on the `shoulda` GitHub page.